packages/react-flow-editor/src/Editor/helpers/getRectFromNode.ts

```diff
@@ -0,0 +1,9 @@
+import { Node } from "@/types"
+
+export const getRectFromNode = (node?: Node): DOMRect => {
```

Comment:
We can write one utility function for this. instead of create two same functions. `getRectFromNode` and `getRectFromRef` looks the same.
```ts
const getDOMRect = (element: Element | null) => {
  const rect = element.getBoundingClientRect()
  
  if (!rect) return new DOMRect()
  
  return rect
}
```